tests/HUBViewControllerTests.m

```diff
@@ -2536,4 +2559,9 @@ - (void)registerAndGenerateComponentsWithNamespace:(NSString *)namespace
     };
 }
 
+- (void)assertEdgeInsetsEqualInsets:(UIEdgeInsets)insets
```

Comment:
Yeah it's a workaround to not being able to use `XCTAssert`s while inside the content inset blocks, since that macro uses self inside 😑  but using another block makes more sense, I'll do that